src/app/services/serviceConfig.js

```diff
@@ -236,6 +241,14 @@ const validate = async (req, currentService, oldService) => {
   unecodedurl = _unescape(postResetUrl);
   postResetUrl = unecodedurl;
   const postUrlValidator = new UrlValidator(postResetUrl);
+  const isPOstResetUrlToLength = await isCorrectLength(postUrlValidator);
```

Comment:
is this PostResetUrlToLength moved above isPostResetUrlValid for requirement?

Reply:
Yes the messages where requested to stack they wanted them to follow the same stacking order